src/Microsoft.DotNet.Tools.Compiler/Program.cs

```diff
@@ -20,6 +20,9 @@ namespace Microsoft.DotNet.Tools.Compiler
 {
     public class Program
     {
+        private const string BuildProfile = "--build-profile";
```

Comment:
I needed the string in multiple places, and I decided to extract it out after getting bitten twice by inconsistent edits.